flow-server/src/main/resources/plugins/application-theme-plugin/theme-generator.js

```diff
@@ -113,23 +111,8 @@ function writeThemeFiles(themeFolder, themeName, themeProperties, options) {
   let filename = basename(styles);
   let variable = camelCase(filename);
 
-  /* LUMO */
-  const lumoImports = themeProperties.lumoImports || ['typography', 'color', 'spacing', 'badge', 'utility'] ;
-  if (lumoImports) {
-    lumoImports.forEach((lumoImport) => {
-      imports.push(`import { ${lumoImport} } from '@vaadin/vaadin-lumo-styles/${lumoImport}.js';\n`);
-      if (lumoImport === 'utility' || lumoImport === 'badge' || lumoImport === 'typography' || lumoImport === 'color') {
-        // Inject into main document the same way as other Lumo styles are injected
-        // Lumo imports go to the theme global imports file to prevent style leaks
-        // when the theme is applied to an embedded component
-        globalFileContent.push(`import '@vaadin/vaadin-lumo-styles/${lumoImport}-global.js';\n`);
-      }
-    });
-
-    lumoImports.forEach((lumoImport) => {
-      // Lumo is injected to the document by Lumo itself
-      shadowOnlyCss.push(`removers.push(injectGlobalCss(${lumoImport}.cssText, '', target, true));\n`);
-    });
+  if(themeProperties.lumoImports) {
+    console.warn('lumoImports is no longer supported in Vaadin 25. To load utility classes, add "importCss": ["@vaadin/vaadin-lumo-styles/utilities.css"]');
```

Comment:
Oops, I made a mistake:

```suggestion
    console.warn('lumoImports is no longer supported in Vaadin 25. To load utility classes, add "importCss": ["@vaadin/vaadin-lumo-styles/utility.css"]');
```